### PR TITLE
Commando csrf

### PR DESCRIPTION
**Pull request by tvergilio**
Some small improvements. Created a new page to allow logged-in users to update their profile. Fixed log out functionality. CSRF should now be possible.

### CODE REVIEW

File: scenarios/security_audit/rand_webapp.xml [hunk 1]
```diff
@@ -98,19 +98,19 @@
     </build>
   </system>
 
-  <system>
```
Comment (cliffe) [inline scenarios/security_audit/rand_webapp.xml:101]: Not sure if you meant to remove Kali from the scenario, or if that's just for testing

**Comment by cliffe** [comment]
Great :)

Not sure if you meant to remove Kali from the scenario, but I'll merge as is, and we can add that back later

File: scenarios/security_audit/rand_webapp.xml (hunk 1 above)
Comment (tvergilio) [reply scenarios/security_audit/rand_webapp.xml:101]: Thanks, Cliffe. I  commented the Kali  provisioning out temporarily as it was causing the build to fail. Error says it was unable to locate the 'kali-tools-web' package. I believe the repository for this dependency may need to be added to the project.

Comment (tvergilio) [reply scenarios/security_audit/rand_webapp.xml:101]: I logged it as ticket #199 as it is a separate issue.

**Pull request merged**